### DISCUSSION TITLE
How make it so i can use my unit layout

### DISCUSSION THREAD

**Discussion by madkaddy** [Vdauphin/HeartsAndMinds#1487] [Question and answer]
hey we are part of a unit that want to use this i have beeing trying to get it so we can spawn in our own sections and fireteams but it does not want to keep the settings it keeps pushing it back to the way you had it. is there a way around this thanks

**Comment by Manthhys** [comment]
Hey  @madkaddy In arma 3 the player slots order in the lobby depends on the chronological order you places the units on the map in the editor. Same applies for groups. You need to manually replace the existing player slots to fit your organisation
